### DISCUSSION TITLE
Please I need help here

### DISCUSSION THREAD

**Discussion by rivate1** [gophish/gophish#3296] [General]
Someone please guide me how to create a perfect phishing campaign using this tool my WhatsApp is +91 78179 70360 anyone please help me 🙏🙏🙏 I'm waiting for your call or text 🙏🙏

**Comment by zaphod13** [comment]
Follow the instructions for adding users, landing pages, sending profiles, etc.  Then add the email templates.  You can find some decent examples  at https://github.com/criggs626/PhishingTemplates .  Then create a campaign, and select the email template, groups, landing page, URL of your goPhish server, sending profile, etc.

**Comment by camilovietnam** [comment]
I have a feeling that this person wants to use this tool to genuinely scam people lol
